codemagic.yaml

```diff
@@ -4,11 +4,12 @@ workflows:
         REPOSITORY: codemagic-ci-cd/cli-tools
     name: Generate binary distribution wheel
     scripts:
       - git tag -a v$(python3 setup.py --version) -m "Release v$(python3 setup.py --version)"
-      - python3 setup.py bdist_wheel
+      - python3 setup.py sdist bdist_wheel
```

Comment:
Why build the source distribution if it's not sent to PyPI nor included in artifacts?

Reply:
Excellent point. Forgot to add the artifacts entry for this.

Reply:
I included tarball artifact in ad2d561.